<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfacing</class>
 <widget class="QMainWindow" name="interfacing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>interfacing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Terminal </string>
     </attribute>
     <widget class="QPlainTextEdit" name="txt_Terminal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>761</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_LocalEcho">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Local Echo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_send">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send !</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txt_kirimstring">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Clear">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Terminal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Graph Plot</string>
     </attribute>
     <widget class="QCustomPlot" name="wgt_GraphPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>841</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Graph</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Table</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_inputdata">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>831</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>851</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>SERIAL UART SETTING</string>
    </property>
    <widget class="QPushButton" name="btn_Open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_portname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>-20</y>
       <width>481</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>FlowControl</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cmb_Parity">
        <property name="currentText">
         <string>No</string>
        </property>
        <item>
         <property name="text">
          <string>No</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Even</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Odd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mark</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>BaudRate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmb_Baud">
        <property name="currentText">
         <string>9600</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pilih</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>DataBits</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>PortName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmb_Port">
        <item>
         <property name="text">
          <string>pilih</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyUSB0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyACM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rfcomm0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyUSB1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyUSB2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyATK0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyATK1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyATK2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyACM0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyACM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyACM2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cmb_StopBits">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="cmb_FlowControl">
        <property name="currentText">
         <string>No</string>
        </property>
        <item>
         <property name="text">
          <string>No</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Software</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="cmb_DataBits">
        <property name="currentText">
         <string>5</string>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>SOLAR CAR ITS RACING TEAM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
   <addaction name="actionAboutQt"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>aboutQt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
